pocs/observatory.py

```diff
@@ -61,9 +63,9 @@ def __init__(self, cameras=None, scheduler=None, dome=None, *args, **kwargs):
         self._image_dir = self.config['directories']['images']
         self.logger.info('\t Observatory initialized')
 
-##########################################################################
-# Helper methods
-##########################################################################
+    ##########################################################################
```

Comment:
Do you have an autoformatter that was indenting these?

Reply:
I think yes, should I stop doing it?

Reply:
We should just make sure it has the same settings that the project uses. We'll worry about it when you're back.